### ISSUE TITLE
Interrogations regarding diploid phenotypes

### ISSUE THREAD

**Issue by PCarme** [pombase/canto#2816]
I just realised that phenotypes annotated on diploid genotypes appear in the "Multi locus genotype annotations and genetic interactions" on the summary page in Canto. 
Also, multi-locus phenotypes can't be annotated as genetic interactions with diploid genotypes. 
Are these normal ?

**Comment by ValWood** [comment]
>I just realised that phenotypes annotated on diploid genotypes appear in the "Multi locus genotype annotations and genetic interactions" on the summary page in Canto.

yes this isn't intuitive, I think there might be a ticket about this that Kim didn't get round to

This might be the ticket, https://github.com/pombase/canto/issues/1932

**Comment by ValWood** [comment]
> Also, multi-locus phenotypes can't be annotated as genetic interactions with diploid genotypes.

THe new GI interface is relatively new and we don't annotate many diploid papers, so I don't think this has come up before.

Probably we will open a separate ticket for this and  keep a note of any sessions this applies to. It will probably not be high priority for a while  because it  will only be a small number of sessions but we can increase the priority if we have a lot. 
I can't remember ever curating a GI  from a diploid!

**Comment by ValWood** [comment]
...even in the old system...

**Comment by PCarme** [comment]
Right, I can imagine this does not happen a lot. Yoshinori's latest community curated paper was ... uncommon ...

**Comment by ValWood** [comment]
@PCarme will describe the diploid genetic interactions from this paper.
@kimrutherford  will try to load them manually (with a script)
We will see what happens.

**Comment by PCarme** [comment]
The publication concerned is PMID:38448160, the curation session is [f0575bfe108d54e1](https://curation.pombase.org/pombe/view/object/curs/6646?model=track). 
After a second glance at the paper, I only see 2 genetic interactions to be annotated. But the way I have annotated the genotypes, it would be in mutants with more than 2 loci affected, like this one :
![image](https://github.com/pombase/canto/assets/163283897/fe7b0cbf-4760-45c6-a392-e5653f47820c)
So, I am guessing that I would first need to change it to only 2 loci. Fortunately, all of the annotations concerned are on genotypes that have two alleles in common (rec8-2A and rec12delta), which I can probably put as background.

**Comment by PCarme** [comment]
The GI to be added are on the two psm3-3A wpl1delta double mutants (in 2 different backgrounds).
![image](https://github.com/pombase/canto/assets/163283897/6a69494d-3531-4581-8f77-79918f086e66)
and
![image](https://github.com/pombase/canto/assets/163283897/c7635a4c-1e18-4650-9535-e9f093e8434a)
 Both are a suppression of the "sister kinetochore dissociation in meiotic metaphase I with equational sister chromatid segregation in meiosis I" phenotypes of the psm3-3A mutants (in each background) by wpl1delta.

**Comment by kimrutherford** [comment]
Hi @PCarme 

What should the interaction type be for the new genetic interactions?

**Comment by kimrutherford** [comment]
Diploid-diploid interactions look OK in Canto when I test on my desktop:

![image](https://github.com/pombase/canto/assets/90474/6ffbeed6-0bc5-4d37-946f-7900bd4e8bb3)

**Comment by PCarme** [comment]
Interaction type should be "Phenotypic suppression" for both. 
In the picture you sent, the phenotype in the "Double mutant phenotype" column is actually the phenotype of the psm3-3A single mutant (with penetrance at 70%)

**Comment by kimrutherford** [comment]
Hi Pascal.  Does this look correct? 

![image](https://github.com/pombase/canto/assets/90474/b2d1576a-9683-4222-a648-c6ec7dff46cd)

![image](https://github.com/pombase/canto/assets/90474/7a4522fb-37cf-406f-994f-e501e0d05c20)

**Comment by PCarme** [comment]
Yes, that's perfect !

**Comment by kimrutherford** [comment]
OK, great.  I will check my code again in the morning and then modify the session in the main Canto.

**Comment by kimrutherford** [comment]
> I will check my code again in the morning and then modify the session in the main Canto.

That's done now.  Let me know if there are any problems.

I've done some testing on my desktop to make sure the website code can handle these interactions.  It doesn't look great, but it seems to work.  Is this display correct?:

![image](https://github.com/pombase/canto/assets/90474/a04c0bf5-de2c-46bd-8d4d-07a8e4f4e658)

**Comment by PCarme** [comment]
It should rather be psm3 rescued by wpl1. Here it appears inverted.

**Comment by kimrutherford** [comment]
No problem.  I have swapped the order of genotypes in the interactions in the Canto session.

**Comment by kimrutherford** [comment]
Hi @PCarme 

https://www.pombase.org/reference/PMID:38448160

This is how it looks on pombase.org.  Is this OK?:

![image](https://github.com/pombase/canto/assets/90474/cb7c0e54-e50e-4f41-adcb-10d3750d9e1c)

**Comment by PCarme** [comment]
That's perfect, thanks Kim !

**Comment by kimrutherford** [comment]
Great.  I think we can close this.
